pages/common/rm.md

```diff
@@ -10,6 +10,10 @@
 
 `rm -r {{/path/to/folder}}`
 
+- Remove file/directory without prompt:
```

Comment:
Yes but that's providing incorrect information to users. We should encourage people to memorize command options by helping them understand what each option means, not by providing formulas to be recalled mechanically and applied blindly as if they were magic incantations.